engine/classes/Elgg/ActionsService.php

```diff
@@ -32,7 +42,7 @@ class ActionsService {
 	 */
 	public function execute($action, $forwarder = "") {
 		$action = rtrim($action, '/');
-		$this->currentAction = $action;
+		$this->current_action = $action;
```

Comment:
Doh. How long has that been that way? Clearly haven't been following it...
(and don't really want to, honestly!)

On Sun, May 31, 2015, 6:24 PM Steve Clay <notifications@github.com> wrote:

> In engine/classes/Elgg/ActionsService.php
> <https://github.com/Elgg/Elgg/pull/8398#discussion_r31397018>:
>
> > @@ -32,7 +42,7 @@ class ActionsService {
> >  	 */
> >  	public function execute($action, $forwarder = "") {
> >  		$action = rtrim($action, '/');
> > -		$this->currentAction = $action;
> > +		$this->current_action = $action;
>
> http://learn.elgg.org/en/1.11/contribute/code.html#naming
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/8398/files#r31397018>.
>